### PR TITLE
Add link to sample repos

### PR DESCRIPTION
**Pull request by faxu**
Changes:
- simplify get started section to directly point to documentation site
- add links to sample repos for inf and training

### CODE REVIEW

**Review by thiagocrepaldi — DISMISSED** [review]
Looks cleaner and we dont have to keep README.md as an index or table of content.

**Comment by jywu-msft** [comment]
can we add a README in the old location: https://github.com/microsoft/onnxruntime/tree/master/samples
pointing people to the new samples repositories?
i've already had customers who used the old samples and couldn't find them anymore.  
(the samples aren't actually merged in the new repo yet)

File: README.md [hunk 1]
```diff
@@ -9,25 +9,14 @@
...
-  * [Execution Providers](https://www.onnxruntime.ai/docs/reference/execution-providers/)
-  * [Releases and servicing](https://www.onnxruntime.ai/docs/reference/releases-servicing.html)
-  * [Citing](https://www.onnxruntime.ai/docs/reference/citing.html)
-* [Additional resources](https://www.onnxruntime.ai/docs/resources/)
+**https://onnxruntime.ai/**
+
+**Usage documention and tutorials** can be found on [onnxruntime.ai/docs](https://onnxruntime.ai/docs).
+
+Companion sample repositories: 
+- ORT Inferencing: [microsoft/onnxruntime-inference-examples](https://github.com/microsoft/onnxruntime-inference-examples)
```
Comment (manashgoswami) [inline README.md:17]: ORT -> ONNX Runtime

File: README.md [hunk 2]
```diff
@@ -9,25 +9,14 @@
...
-  * [Deploy on mobile](https://www.onnxruntime.ai/docs/how-to/deploy-on-mobile.html)
-  * [Use custom ops](https://www.onnxruntime.ai/docs/how-to/add-custom-op.html)
-  * [Add a new EP](https://www.onnxruntime.ai/docs/how-to/add-execution-provider.html)
-* [Reference](https://www.onnxruntime.ai/docs/reference)
-  * [API documentation](https://www.onnxruntime.ai/docs/reference/api/)
-  * [Execution Providers](https://www.onnxruntime.ai/docs/reference/execution-providers/)
-  * [Releases and servicing](https://www.onnxruntime.ai/docs/reference/releases-servicing.html)
-  * [Citing](https://www.onnxruntime.ai/docs/reference/citing.html)
-* [Additional resources](https://www.onnxruntime.ai/docs/resources/)
+**https://onnxruntime.ai/**
```
Comment (manashgoswami) [inline README.md:12]: Add a sentence about "Go to .....<link>"

**Review by snnn — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (faxu) [reply README.md]: Updated

File: README.md (hunk 1 above)
Comment (faxu) [reply README.md]: Updated

**Comment by faxu** [comment]
> can we add a README in the old location: https://github.com/microsoft/onnxruntime/tree/master/samples
> pointing people to the new samples repositories?
> i've already had customers who used the old samples and couldn't find them anymore.
> (the samples aren't actually merged in the new repo yet)

Added

**Review by jywu-msft — APPROVED** [review]

**Review by thiagocrepaldi — APPROVED** [review]

**Pull request merged**